### DISCUSSION TITLE
Artists and Album Titles of local mp3 files reference real Artists page

### DISCUSSION THREAD

**Discussion by weshford** [dargmuesli/spotitag#18] [Ideas]
I'm still trying to find out, how exactly Spotify stores an artists on a file. 
For example, when listening to a song, you can always click on the artists on under the song title. Spotify then redirects you to the artists page. This even works when offline, although nothing like profile picture etc is loaded, but it does keep track of how many songs by that artists are saved in your "Liked Songs"

With local files, a press on the artists justs pops up the message box ("Couldn't Open Link. Spotify can't open this type of link on this device.")

There has to be some way, to manipulate an id3 Tag (with version 2.4) so that local files can. I have not been able to recreate it yet.

**Comment by dargmuesli** [comment]
I think this question could well be asked on StackOverflow, as this project is not about their client.

**Reply by weshford** [reply]
You're right sorry! Just wanted to share it here directly to the creator as well as an idea propose!
